app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -752,7 +753,7 @@ protected void showStreamItemDialog(final PlaylistStreamEntry item) {
 
         final ArrayList<StreamDialogEntry> entries = new ArrayList<>();
 
-        if (PlayerHolder.getInstance().getType() != null) {
+        if (PlayerHolder.getInstance().isPlayerOpen()) {
             entries.add(StreamDialogEntry.enqueue);
```

Comment:
Also show it in the local playlist fragment
```suggestion
            entries.add(StreamDialogEntry.enqueue);

            if (PlayerHolder.getInstance().getQueueSize() > 1) {
                entries.add(StreamDialogEntry.next);
            }
```